### ISSUE TITLE
Delete "Retention rules" log

### ISSUE THREAD

**Issue by Qercoat** [goharbor/harbor#19466]
Is it possible to add a log for removing "Retention rules"?
Currently there is information about an update, not a deletion, and it is in the DEBUG log.

`Oct 18 10:17:24 IP core[5950]: 2023-10-18T08:17:24Z [DEBUG] [/pkg/permission/evaluator/admin/admin.go:35]: system administrator require update action for resource /project/94/tag-retention`

If it is not possible to add a log on deletion, is it possible to move this log from DEBUG to INFO?

**Comment by wy65701436** [comment]
let me clarify, you would like to add audit logs for retention operations, right?

**Comment by Qercoat** [comment]
Yes, currently in the DEBUG logs there is information about a retention policy update.

Is it possible for there to be logs for policy removal the same as for project removal?
`
Oct 18 07:43:52 66a982913a44 audit[1]: 2023-10-18T07:43:52Z [INFO] [/pkg/audit/manager.go:76][operator="" resourceType="project" time="2023-10-18 07:43:52.72521725 +0000 UTC m=+1256.975537422"]: action:delete, resource:1`

Or changing the log level about the update from DEBUG to INFO?

**Comment by Qercoat** [comment]
Is it possible to do this?
